src/js/ui/header/header.js

```diff
@@ -17,10 +17,10 @@ export const header = () => {
   logoContainer.href = '/';
 
   const img = document.createElement('img');
-  img.classList.add('Logo-noroff', 'my-auto', 'text-body');
+  img.classList.add('my-auto');
   img.src = '/assets/icons/noroff-logo.svg';
-  img.setAttribute('width', '150');
-  img.setAttribute('height', '50');
+  img.setAttribute('width', '200');
+  img.setAttribute('height', '100');
```

Comment:
This looks SIGNIFICANTLY better, but could you change the height attribute to be 70 instead of 100, due to the navbar becoming too large when it's at 100. :D Otherwise good job!

Reply:
Maybe even 60 to closer resemble the original in size

Reply:
I had that at first, but increased the sized because now since the words are in the svg and not in seperate div's the text shrinks 
a bit. I will change it to 60 and then if we decided to change it later we can.